README.md

```diff
@@ -30,6 +30,22 @@ Currently we are targeting five platforms: OS X, Ubuntu Linux, iOS, Android, and
 
 ## platforms ##
 
+### Windows (command line) ###
+Downloading angle https://www.dropbox.com/s/vollhh52bjemeym/angle.7z?dl=0 and extract to `build` directory
```

Comment:
Well actually the Servo fork only compiles the HLSL translator :\ But this repo has a full CMake build script! https://github.com/solvespace/angle/tree/chromium/2704+cmake